src/rules/memberOrderingRule.ts

```diff
@@ -396,6 +396,7 @@ function getOptionsJson(allOptions: any[]): { order: MemberCategoryJson[], alpha
         return { order: convertFromOldStyleOptions(allOptions), alphabetize: false }; // presume allOptions to be string[]
     }
 
+    // tslint:disable-next-line no-unsafe-any
     return { order: categoryFromOption(firstOption[OPTION_ORDER]), alphabetize: !!firstOption[OPTION_ALPHABETIZE] };
```

Comment:
instead of disabling, annotate `firstOption` as type `Options`, then call using `firstOption.order` and `firstOption.alphabetize`